website/docs/cli/configuration/markdown-styling.mdx

```diff
@@ -0,0 +1,298 @@
+---
+title: Markdown Styling
+sidebar_label: Markdown Styling
+sidebar_position: 7
+description: Configure custom markdown styling for Atmos CLI output
+---
+
+# Terminal Styling Configuration
+
+Atmos provides a unified styling system for all terminal output. This includes documentation, command help text, error messages, and syntax highlighting.
+
+## Configuration
+
+The styling is configured in your `atmos.yaml` file under the `settings.terminal` section:
+
+```yaml
+settings:
+  terminal:
+    # Color theme configuration
+    theme:
+      # Base colors used throughout the application
+      colors:
+        primary: "#00A3E0"    # Atmos blue
+        secondary: "#9B51E0"  # Purple
+        success: "#48BB78"    # Green
+        warning: "#ECC94B"    # Yellow
+        error: "#F56565"      # Red
+        text: "#FFFFFF"       # White
+        muted: "#4A5568"      # Gray
+      
+      # Dark mode variants
+      dark:
+        text: "#F7FAFC"      # Off white
+        muted: "#A0AEC0"     # Light gray
+        background: "#1A202C" # Dark slate
+
+    # Documentation and help text styling
+    docs:
+      max_width: 120  # Maximum width for documentation
+      pager: true     # Use pager for long output
+      
+      # Markdown element styling
+      markdown:
+        document:
+          color: "${colors.text}"
+        heading:
+          color: "${colors.primary}"
+          bold: true
+        code_block:
+          color: "${colors.secondary}"
+          margin: 1
+        link:
+          color: "${colors.primary}"
+          underline: true
+        strong:
+          color: "${colors.secondary}"
+          bold: true
+        emph:
+          color: "${colors.muted}"
+          italic: true
+
+    # Syntax highlighting configuration
+    syntax:
+      enabled: true
+      theme: "dracula"      # or use custom colors from theme
+      line_numbers: true
+      wrap: false
+      
+    # Console output configuration
+    console:
+      pager: true           # Enable pager for long output
+      timestamps: false     # Show timestamps in logs
+      colors: true         # Enable colored output
+      unicode: true        # Use unicode characters
```

Comment:
`settings.console` and `settings.terminal` are redundant. Just consolidate underneath `settings.terminal`